doc/HACKING/Maintaining.md

```diff
@@ -0,0 +1,115 @@
+# Maintaining Tor
+
+This document details the roles and processes on maintaining the Tor code
+base. We do have many policies in place and the maintainers are there as a
+last line of defense to enforce them.
```

Comment:
IMO this paragraph is a little iffy.  I don't agree that the main role of the maintainer is to defend against bad code.  We're trying to make a better Tor, but that happens not only by rejecting things that are plainly noncompliant. We have to encourage people to get better, and try to help every release of Tor to be better than the one before.

I think our job isn't rejecting; it's oversight and teaching.

Reply:
Not sure this paragraph mentions "bad code" or anything about rejecting? ... But it is rather to enforce our code standard policies which is the "oversight" there that you mention.

Reply:
Fixup commit to remove that last sentence: 633135f0b9